dax-data-set-descriptors/jfk.yaml

```diff
@@ -72,3 +72,9 @@ related_assets:
     description: Elyra example pipeline
     mime_type: text/html
     url: https://github.com/elyra-ai/examples/tree/master/pipelines/dax_noaa_weather_data
+  - name: OpenAIHub notebook
+    description: OpenAIHub exmple notebook with data mount
+    mime_type: application/yaml
+    url: https://github.ibm.com/OpenAIHub/openaihub/blob/master/notebooks/samples/JFK-airport.yaml
+#    application: OpenAIHub
```

Comment:
why are we commenting this, for testing purpose?

Reply:
yup, my mistake, I updated it